tests/fixtures/moleculer/get.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Context, GenericObject } from 'moleculer';
+import { Action, Method, Service } from 'moleculer-decorators';
+import { Get, Ignore, Path, Route, Security } from '../../../src';
+import { ModelService } from '../services/modelService';
+import { TestClassModel } from '../testModel';
+import { User } from './api.service';
+const moleculer = require('moleculer');
+interface ChatsActionParams {
+  withUser: string;
+}
+
+export interface AuthMeta {
+  user: User;
+  $statusCode?: number;
+}
+
+export interface AuthContext<P = GenericObject> extends Context<P, AuthMeta> {
+  meta: AuthMeta;
+  params: P;
+}
+
+@Service()
+@Route('GetTest')
+class GetTest  extends moleculer.Service  {
+  @Action({
+    params: {
+      withUser: 'string',
+    },
+  })
+  public async getModel(ctx: AuthContext<ChatsActionParams>) {
+    const { withUser } = ctx.params;
+    const fromUser = ctx.meta.user.id;
+    return this._getModel(withUser, fromUser);
+  }
+
+  @Get('/getModel/{withUser}')
+  @Security('ApiKeyAuth')
+  @Method
+  private _getModel(@Path() withUser: string, @Ignore() fromUser: string): Promise<TestClassModel> {
```

Comment:
I don't see `fromUser` in the path of the `@Get` decorator. So it's ignoring a path variable that isn't on the path. So this test is not set up correctly.